File: Examples/test-suite/ruby/extend_template_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# Author::    gga
# Copyright:: 2007
# License::   SWIG
#

require 'swig_assert'

require 'extend_template'

f = Extend_template::Foo_0.new
if f.test1(37) != 37
  raise RuntimeError
end

if f.test2(42) != 42
  raise RuntimeError
end

File: Examples/test-suite/javascript/extend_template_runme.js
var extend_template = require("extend_template");

var f = new extend_template.Foo_0();
if (/* await */(f.test1(37)) != 37) {
    throw new Error;
}

if (/* await */(f.test2(42)) != 42) {
    throw new Error;
}
